src/app/components/search-bar/search-bar.component.ts

```
@@ -225,6 +239,10 @@ export class SearchBarComponent implements OnInit, OnChanges{
     return this.searchParameters?.filter((param: any) => param.schema?.category_id === category);
   }
 
+  isParameterDisabled(parameterName: string): boolean {
+    return this.extraParameters && this.extraParameters[parameterName];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)